src/main/java/muramasa/gti/common/tileentities/multi/TileEntityPrimitiveBlastFurnace.java

```diff
@@ -10,9 +10,9 @@
     public boolean onStructureFormed() {
         int3 controller = new int3(getPos(), getFacing());
         controller.back(1);
-        getWorld().setBlockState(controller.asBP(), Blocks.LAVA.getDefaultState(), 3);
+        getWorld().setBlockState(controller.toImmutable(), Blocks.LAVA.getDefaultState(), 3);
         controller.up(1);
-        getWorld().setBlockState(controller.asBP(), Blocks.LAVA.getDefaultState(), 3);
+        getWorld().setBlockState(controller.toImmutable(), Blocks.LAVA.getDefaultState(), 3);
```

Comment:
Can you commit this in a separate branch and open a separate PR?

Reply:
Can you explain more detailed? I see only one branch for Gregtech. Will I have access to do pull in a new branch?

Reply:
No... _you_ create a new branch, and PR into our _master branch_

Reply:
Ok, then close this PR. Is I should create a new branch for all  3 projects to commit?

Reply:
No, just revert the last commit, the one where you commited the lines I highlighted here. Leave this open as this is something separate, then commit these lines in another branch and open a separate PR.